lib/matplotlib/streamplot.py

```diff
@@ -52,6 +52,8 @@ def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
         See :class:`~matplotlib.patches.FancyArrowPatch`.
     *minlength* : float
         Minimum length of streamline in axes coordinates.
+    *start_points*: Nx2 array
+        Coordinates of starting points for the streamlines. In array coordinates.
```

Comment:
oh yeah, that's an oversight, I have not modified that since I worked out the data-array coordinate conversion.